include/deal.II/lac/precondition.h

```diff
@@ -2297,25 +2325,28 @@ PreconditionChebyshev<MatrixType, VectorType, PreconditionerType>::
 
       // read the eigenvalues from the attached eigenvalue tracker
       if (eigenvalue_tracker.values.empty())
-        min_eigenvalue = max_eigenvalue = 1;
+        info.min_eigenvalue = info.max_eigenvalue = 1.;
       else
         {
-          min_eigenvalue = eigenvalue_tracker.values.front();
+          info.min_eigenvalue = eigenvalue_tracker.values.front();
 
           // include a safety factor since the CG method will in general not
           // be converged
-          max_eigenvalue = 1.2 * eigenvalue_tracker.values.back();
+          info.max_eigenvalue = 1.2 * eigenvalue_tracker.values.back();
```

Comment:
Should this factor be reflected in the documentation somewhere? It means that even if you run the solver to convergence, you'll get an overestimate by a factor of 1.2...